modules/apps/calendar/calendar-test/src/testIntegration/java/com/liferay/calendar/upgrade/v4_2_1/test/UpgradeCalendarBookingTest.java

```diff
@@ -55,10 +70,10 @@ public void setUp() throws Exception {
 		_upgradeDatabaseTestHelper =
 			CalendarUpgradeTestUtil.getUpgradeDatabaseTestHelper();
 		_upgradeProcess = CalendarUpgradeTestUtil.getUpgradeStep(
-			_upgradeStepRegistrator,
-			"com.liferay.calendar.internal.upgrade.v4_2_1." +
-				"CalendarBookingUpgradeProcess");
+			_upgradeStepRegistrator, _CLASS_NAME);
 		_user = UserTestUtil.addUser();
+
+		_serviceContext = createServiceContext();
```

Comment:
@rijema make `createServiceContext` private

Reply:
As I sent to an util class, I had to leave it public. Is there any problem?